generated/nodejs/google-cloud-node/packages/speech/src/v1beta1/speech_api.js

```diff
@@ -208,6 +211,35 @@ SpeechApi.prototype.asyncRecognize = function(request, options, callback) {
   return this._asyncRecognize(request, options, callback);
 };
 
+/**
+ * Perform bidirectional streaming speech-recognition: receive results while
+ * sending audio. This method is only available via the gRPC API (not REST).
+ *
+ * @param {Object=} options
+ *   Optional parameters. You can override the default settings for this call, e.g, timeout,
+ *   retries, paginations, etc. See [gax.CallOptions]{@link https://googleapis.github.io/gax-nodejs/global.html#CallOptions} for the details.
+ * @returns {Stream}
+ *   An object stream which is both readable and writable. It accepts objects
+ *   representing [StreamingRecognizeRequest]{@link StreamingRecognizeRequest} for write() method, and
+ *   will emit objects representing [StreamingRecognizeResponse]{@link StreamingRecognizeResponse} on 'data' event asynchronously.
+ *
+ * @example
+ *
+ * var api = speechV1beta1.speechApi();
+ * var stream = api.streamingRecognize().on('data', function(response) {
+ *     // doThingsWith(response);
+ * });
+ * var request = {};
+ * // Write request objects.
+ * stream.write(request);
```

Comment:
Oh, right. I've been doing imperative languages for too long. :)

How about producing streaming requests?